packages/kit/src/exports/vite/index.js

```diff
@@ -322,11 +308,9 @@ async function kit({ svelte_config }) {
 						//    when it is detected to keep our virtual modules working.
 						//    See https://github.com/sveltejs/kit/pull/9172
 						//    and https://vitest.dev/config/#deps-registernodeloader
-						'@sveltejs/kit',
-						// We need to bundle any packages depending on @sveltejs/kit so that
-						// everyone uses the same instances of classes such as `Redirect`
-						// which we use in `instanceof` checks
-						...packages_depending_on_svelte_kit
+						'@sveltejs/kit'
+						// Don't bundle packages that depend on `@sveltejs/kit` because they
+						// may include CJS syntax/dependencies
```

Comment:
@eltigerchino did you intend to leave this comment here?

Reply:
Yeah, just as a reminder for the future. It doesn't have to be there though.